src/include/sof/spinlock.h

```diff
@@ -83,15 +83,8 @@
 extern uint32_t lock_dbg_atomic;
 extern uint32_t lock_dbg_user[DBG_LOCK_USERS];
 
-/* all SMP spinlocks need init, nothing todo on UP */
-#define spinlock_init(lock) \
-	do { \
-		arch_spinlock_init(lock); \
-		(*lock)->user = __LINE__; \
-	} while (0)
-
 /* panic on deadlock */
-#define spin_try_lock_dbg(lock) \
+#define spin_try_lock_dbg(lock, line) \
```

Comment:
This isn't very logical, I think. You add a parameter but you don't use it. You can convert `__LINE__` below to `line` now, or maybe you can convert this macro to an inline function now.

Reply:
This macro will remain a macro. The fact that there is `__LINE__` below comes from me not being able to track all changes in a rebase conflict, I need to fix it.